### ISSUE TITLE
Can animate with react-native-reanimated 3?

### ISSUE THREAD

**Issue by brugali-anoki** [lottie-react-native/lottie-react-native#1014]
Hi, i have a app with:

    "lottie-react-native": "^6.0.0-rc.3", // or 5.1.5, is indifferent for me
    "react-native-reanimated": "^3.0.2",

I need to animate my animation like this:
When I scroll through my flatlist, the animation should move accordingly

I tried with

```
import Animated, {
  Extrapolation,
  useAnimatedScrollHandler,
  useAnimatedStyle,
  useSharedValue,
  interpolate,
  interpolateColor,
  useAnimatedProps,
  withTiming,
} from 'react-native-reanimated';

export const TutorialScreen = ({ navigation }) => {

  const Lottie = Animated.createAnimatedComponent(AnimatedLottieView);
  
  const scrollX = useSharedValue(0);
  const scrollHandler = useAnimatedScrollHandler({
    onScroll: (e) => {
      scrollX.value = e.contentOffset.x;
    },
  });

  const animatedProps = useAnimatedProps(() => {
    const progress = interpolate(scrollX.value, [0, WIDTH_DEVICE * 2], [0, 1], { extrapolateRight: Extrapolation.CLAMP });
    return {
      progress: progress,
    };
  });


return (
<View>
   <Animated.FlatList
        ref={flatListRef}
        viewabilityConfigCallbackPairs={viewabilityConfigCallbackPairs.current}
        horizontal={true}
        style={styles.flatList}
        contentContainerStyle={{
          height: '100%',
        }}
        data={tutorialTabs}
        renderItem={({ item, index }) => <TutorialStep index={index} slide={item} scrollX={scrollX} />}
        snapToInterval={WIDTH_DEVICE}
        snapToAlignment="start"
        bounces={false}
        windowSize={10}
        initialNumToRender={9}
        decelerationRate={isAndroid ? 'fast' : 0}
        showsHorizontalScrollIndicator={false}
        keyExtractor={(userBook) => userBook.id.toString()}
        onScroll={scrollHandler}
        scrollEventThrottle={16}
      />
              <Lottie source={LottieTutorial} style={styles.animation} pointerEvents={'none'} animatedProps={animatedProps} />

</View>
)
```

But not work

**Comment by matinzd** [comment]
Just released `6.0.0-rc.4`. Please try again and let me know.

**Comment by matinzd** [comment]
Duplicate of https://github.com/lottie-react-native/lottie-react-native/issues/1003#issuecomment-1517671001.
